torch/csrc/jit/import.cpp

```diff
@@ -411,6 +411,10 @@ std::shared_ptr<script::Module> load(std::istream& in) {
 std::shared_ptr<script::Module> load(const std::string& filename) {
   std::ifstream in(filename, std::ios_base::binary);
 
+  if (in.fail()) {
```

Comment:
Any reason you can't AT_CHECK this?

Reply:
No, I just always try to blend in locally and there is a throw std::runtime_error in that file but no AT_CHECK. I'll change it to AT_CHECK.

Reply:
Thank you :)